packages/button/src/test/mwc-button.test.ts

```diff
@@ -22,51 +22,92 @@ import {fixture, rafPromise, TestFixture} from '../../../../test/src/util/helper
 
 const ICON_SELECTOR = '.mdc-button__icon';
 
-const noText = html`<mwc-button></mwc-button>`;
-
-const basic = html`
-  <mwc-button>this is a button</mwc-button>
-`;
+interface ButtonProps {
+  disabled: boolean;
+  unelevated: boolean;
+  raised: boolean;
+  dense: boolean;
+  outlined: boolean;
+  text: string;
+  label: string;
+  icon: string;
+  trailingIcon: boolean;
+}
+
+const button = (propsInit?: Partial<ButtonProps>) => {
+  if (!propsInit) {
```

Comment:
I think instead of adding this to the generator, this should simply be a different, static TemplateResult. i.e. the old fashioned way

e.g.

```TS
buttonBasic = html`<mwc-button></mwc-button>`;
```